firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -318,7 +318,7 @@ extension BrowserViewController: ToolBarActionMenuDelegate {
         presentWithModalDismissIfNeeded(viewController, animated: true)
     }
 
-    func showToast(message: String, toastAction: MenuButtonToastAction) {
+    func showToast(_ bookmarkURL: URL? = nil, _ title: String?, message: String, toastAction: MenuButtonToastAction) {
         switch toastAction {
         case .removeBookmark:
             let viewModel = ButtonToastViewModel(labelText: message,
```

Comment:
```suggestion
        case .removeBookmark:
            guard let bookmarkURLString = bookmarkURL?.absoluteString else { return }

            let viewModel = ButtonToastViewModel(labelText: message,
```


You could add this and remove some of the optional checks further down (I think if we can't get the url being removed here, no need to run the rest of the code)